### PR TITLE
Brazilian portuguese translation

### PR DESCRIPTION
**Pull request by atilacamurca**

### CODE REVIEW

**Comment by atilacamurca** [comment]
I did the translation using the web tool <https://crowdin.com/project/mgba-pt-br>.

I think it would be interest if the mgba project had its own page. It's free for open source projects and it's easy to get started. You just create a project page and submit the `.ts` files.

**Comment by endrift** [comment]
I've been trying to decide between Crowdin, Transifex and Weblate before committing to one platform, but I have been meaning to use one.

I'm going to wait for someone I know who knows pt_BR to review this before I merge it though.

**Comment by atilacamurca** [comment]
Great! Thanks for the response. I understand we have to be careful with those type of contributions.

Two things I want to ask is:

- To have a template `.ts` file for those who are starting to translate. That is, an  empty file with only the english string. The crowdin tool was able to do that for me, but it would be nice for those using other tools
- That the file have guiding comments for more context. For example, I saw in other translations that people translate the word **English** instead of putting his own language. I could see that based on other translations that I did, but it was not clear.

**Comment by lotharsm** [comment]
@endrift somewhat unrelated - but if you decide to use a translation system, I'd always prefer Weblate over Transifex (I simply can't stand Transifex, it feels so clumsy...)

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:1664]: "Modo dos planos de fundo"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:1674]: "rotacionado/escalonado" pra ser consistente com a linha abaixo.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:1719]: "Forçar limpeza da tela"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:1794]: "Scanline do VCounter"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:1905]: "inteira" minúscula pra ser consistente com as outras linhas.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:1919]: "X final", "X inicial", etc.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:2072]: "Tamanho vertical do mosaico de objeto", pra ser consistente. (Linha abaixo também.)

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:2127]: "Clarear"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:2132]: "Escurecer"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:3120]: "Erro do Jogo"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:3526]: Acho que não se traduz "Shader", mas senão, acho que "Filtros" seria mais adequado.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:3544]: etc. Acho que "sombreamento" fica confuso.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:3614]: "arquivo de vídeo de saída"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:3837]: "Rodar BIOS" acho que fica melhor

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:3963]: "Remover game pak"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:4073]: "Fixar proporção" ou "Travar proporção"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:4460]: "Horário iniciando em"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:4470]: `dd/MM/yy hh:mm:ss` pra ficar no formato brasileiro.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:4653]: "Volume durante avanço rápido:"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:4658]: "Driver de exibição:"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:4704]: Novamente, "fixar" ou "travar" fica melhor eu acho.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:4912]: "conhecido**s**"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:5033]: "Arquivos de salvamento"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:5102]: "Cores BG padrão:" ou "Cores padrão do BG:"

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:5130]: "Filtros" etc.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:5402]: "Sem compressão"

**Review by yuriks — CHANGES_REQUESTED** [review]
I found a few lines that sound incorrect in context, so left suggestions for those. I also left suggestions for several other things where I thought the wording or word choice could be improved.

Thanks for doing the translation!

**Comment by yuriks** [comment]
(Note: I marked my review "Request changes" but it's up to @endrift whether to merge or not. Most of it looks fine as-is.)

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (atilacamurca) [reply src/platform/qt/ts/mgba-pt_BR.ts:663]: Obrigado @yuriks pelas sugestões. Algumas string eu deixei de propósito sem tradução devido a dificuldade de entendimento por parte do usuário. Vou revisar os arquivos com as sugestões e logo darei novo retorno.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (atilacamurca) [reply src/platform/qt/ts/mgba-pt_BR.ts:3544]: @yuriks o que você acha de não traduzirmos shader? Em caso positivo posso trocar logo todas as ocorrências.

Comment (yuriks) [reply src/platform/qt/ts/mgba-pt_BR.ts:3544]: Acho válido, e provavelmente uma boa ideia pra ser menos ambíguo. Pra falar a verdade não costumo usar software traduzido pra Português muito, então não tinha certeza do jeito comum de traduzir "shader" ou se geralmente é só mantido cono palavra estrangeira.

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (yuriks) [inline src/platform/qt/ts/mgba-pt_BR.ts:1266]: This line should be updated with "Não licenciado" to match the one before.

**Review by yuriks — APPROVED** [review]
Really sorry for taking a while to get back to this. I spotted one line that wasn't updated, but other than that it looks good to merge to me!

File: src/platform/qt/ts/mgba-pt_BR.ts
Comment (atilacamurca) [reply src/platform/qt/ts/mgba-pt_BR.ts]: I just updated this translation, thank you.

**Review by yuriks — APPROVED** [review]

**Comment by atilacamurca** [comment]
Thank you @yuriks for the review and suggestions.

And thank you @endrift and all contributors of the project.

**Pull request merged**
